python/dgl/graphbolt/graph_storage/csc_sampling_graph.py

```diff
@@ -227,6 +228,9 @@ def sample_neighbors(
             Boolean indicating whether the sample is preformed with or
             without replacement. If True, a value can be selected multiple
             times. Otherwise, each value can be selected only once.
+        return_eids: bool
+            Boolean indicating whether edge IDs need to be returned, typically
+            used when edge features are required.
```

Comment:
The doc does not describe what's the return type.

Reply:
Added